### PR TITLE
Bump @metamask/assets-controllers version to v19.0.0

### PR DESCRIPTION
**Pull request by adonesky1**
Bump @metamask/assets-controllers version to v19.0.0

### CODE REVIEW

**Comment by adonesky1** [comment]
@metamaskbot policies-update

**Comment by adonesky1** [comment]
@metamaskbot update-policies

**Review by BelfordZ — DISMISSED** [review]

File: test/e2e/tests/nft/erc1155-interaction.spec.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ describe('ERC1155 NFTs testdapp interaction', function () {
 
   it('should enable approval for a third party address to manage all ERC1155 token', async function () {
     const expectedMessageTitle =
-      'Allow access to and transfer all of your NFTs from this collection?';
+      'Allow access to and transfer all of your NFTs from ERC1155?';
```
[the remark this answers is not in the document]
Comment (jiexi) [reply test/e2e/tests/nft/erc1155-interaction.spec.js:135]: worth leaving a comment? This may cause confusion for others in the future

**Review by jiexi — APPROVED** [review]
LGTM. feel free to ignore the nit above

File: test/e2e/tests/nft/erc1155-interaction.spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (adonesky1) [reply test/e2e/tests/nft/erc1155-interaction.spec.js:135]: I've added a comment on line 133

**Review by legobeat — APPROVED** [review]
lgtm

**Pull request merged**